docs/docs/basic-hardware-software-requirements.md

```diff
@@ -2,6 +2,34 @@
 title: Basic Hardware and Software Requirements
 ---
 
-This is a stub. Help our community expand it.
+Find the latest hardware and software requirements for building with Gatsby. Note: this doc will evolve as the framework evolves, so if you find something that's outdated please [open an issue with your findings](/contributing/how-to-file-an-issue/).
 
-Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your pull request gets accepted.
+## Operating Systems
+
+- Mac OSX Sierra or later (10.12)
+- Windows 10 ([setup instructions](/docs/gatsby-on-windows/))
+- Linux ([multiple distributions](/docs/gatsby-on-linux/))
+
+## Memory
+
+This will be variable depending on the size of your site. Gatsby sites have been known to work from 500mb to 1GB of RAM.
+
+## Node.js ecosystem
+
+To develop with Gatsby, you'll need to install:
+
+- Node 10.13.0 (LTS) or higher
+  - See [Gatsby notice of dropping Node 8](/blog/2020-03-20-dropping-support-for-node-8)
```

Comment:
Not sure that this link would be valuable 🤔 We'll be constantly dropping support for older versions, so would this be always replaced or a long list of posts?

Reply:
We could add onto it? I find the reasoning to be useful. We'll likely have to update the version anyway when the next post comes out.

Reply:
But I don't really see the benefit of adding it vs the cognitive load the read and/or maybe click on that link. When you look at other "min requirements" docs you won't see information about dropping support for version X as that information is obsolete anyways.

Reply:
It might be of interest to people reading this doc but I agree that it could potentially add unnecessary cognitive load. I'd leave it out, at least for now.

Reply:
Ok, thanks! I pushed an update to remove that line.